esmvaltool/diag_scripts/droughts/pet.R

```diff
@@ -0,0 +1,223 @@
+#' @importFrom checkmate makeAssertCollection
+#' @importFrom stats cycle end frequency start ts optim
+#' @importFrom zoo as.yearmon rollapply
+
+# This file is based on code from diag_save_spei_all.R, but only contains the
+# parts to calculate PET. The idea of calculating PET in a seperate diagnostic
+# is to be more flexible in the choice of PET calculation methods. It saves
+# the calculated PET with correct units and metadata to be used as an ancestor
+# for diag_spei.R or diag_spei.py, but also allows to use pet as a variable from
+# a preprocessed dataset in the same way.
+#
+# Some functions that are used by diag_spei.R too are moved to a shared utils.R
+#
```

Comment:
```suggestion
```
Same with this - make sure it's in the changelog, maybe in the documentation. Don't think it needs to be in this file.